src/main/java/com/auth0/utils/tokens/IdTokenVerifier.java

```diff
@@ -255,26 +255,26 @@ public Builder withLeeway(Integer leeway) {
         }
 
         /**
```

Comment:
I moved the `withOrganization` method closer to the other public builder methods for readability/organization purposes. The `withClock` method is not removed, the only functional change is making the `withOrganization` method `public`.